webrtc.html

```diff
@@ -8999,7 +9000,8 @@ <h2>
                 </p>
                 <ol class=algorithm id="setparameters-algo">
                   <li class="no-test-needed">Let <var>parameters</var> be the
-                  method's first argument.
+                  method's first argument and let <var>setParameterOptions</var> be the second
+                  argument which is ununsed in this specification and shall be ignored.
```

Comment:
I think it's better to not mention the second argument here, and not say anything that suggests it must be ignored.
```suggestion
                  method's first argument.
```